libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/headerSection.spec.ts

```diff
@@ -0,0 +1,110 @@
+const layoutRow_02 = new LayoutRow(1);
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+describe("Header section", { tags: ["pre-merge_isolated_tiger"] }, () => {
+    describe("Default language", () => {
+        beforeEach(() => {
+            cy.login();
+
+            Navigation.visit("dashboard/header");
+            editMode.isInEditMode();
+            insightCatalog.waitForCatalogReload();
+        });
+
+        it("(SEPARATE) can update header for all sections", () => {
+            let i = 0;
+            let t = 0;
+            cy.fixture("headerDataTest").then((data) => {
+                data["DataTest"].forEach((result: { sectionName: string }) => {
+                    new LayoutRow(i)
+                        .getHeader()
+                        .setTitle(result.sectionName)
+                        .setDescription(result.sectionName);
+                    i += 1;
+                });
+            });
+
+            editMode.save(); //save all headers after input
+
+            cy.fixture("headerDataTest").then((data) => {
+                data["DataTest"].forEach((result: { sectionName: string }) => {
+                    if (t === 0) {
+                        new LayoutRow(t).getHeader().hasTitle(false).hasDescription(false);
+                    } else {
+                        new LayoutRow(t)
+                            .getHeader()
+                            .hasTitleWithText(result.sectionName)
+                            .hasDescriptionWithText(result.sectionName);
+                    }
+                    t += 1;
+                });
+            });
+        });
+
+        it("Header is removed after latest insight is deleted from a section", () => {
+            layoutRow_01.hasCountOfHeaderSection(3);
+            new Widget(0).removeVizWidget();
+
+            layoutRow_01.hasCountOfHeaderSection(2);
+        });
+    });
+
+    describe("Localization", () => {
+        before(() => {
+            cy.login();
+
+            Navigation.visit("dashboard/header-localization");
+            //cy.wait(10000); // need time to load localization config
+            editMode.isInEditMode();
+            insightCatalog.waitForCatalogReload();
+        });
+        it("Header placeholder should be translated", () => {
+            layoutRow_01
+                .getHeader()
+                .setTitle(" ")
+                .setDescription(" ")
+                .hasPlaceholderTitle("Ajouter un titre ici...")
+                .hasPlaceholderDescription("Ajouter une description ici...");
+        });
+
+        it("(SEPARATE) limitation of title, description", () => {
+            cy.fixture("headerDataTest").then((data) => {
+                const title = data["LimitTexts"].title;
```

Comment:
and question again, why do you need to use fixture json here? title has only one value, description has only one value, you can define these two strings as global constant of this file. Only need to use fixture json if this is a data-driven test

Reply:
there is no meaning to put these strings to a separate json file

Reply:
the data test is too long, input directly in the test spec file will make mess up